plugin.rb

```diff
@@ -18,6 +18,7 @@
   require_dependency 'admin_constraint'
 
   module ::Patreon
+    PLUGIN_NAME = 'discourse-patreon'.freeze
```

Comment:
`PLUGIN_NAME` is already defined here: https://github.com/discourse/discourse-patreon/blob/master/plugin.rb#L12

Why define again?

Reply:
We are defining `PLUGIN_NAME` in many places. For example [#1](https://github.com/discourse/discourse-patreon/blob/master/app/controllers/patreon_admin_controller.rb#L5), [#2](https://github.com/discourse/discourse-patreon/blob/master/lib/pledges.rb#L5) and [#3](https://github.com/discourse/discourse-patreon/blob/master/lib/seed.rb#L3). Because in `plugin.rb` it was defined in outside of module. So we can't access it anywhere else.

After the issues are sorted I am going to replace everywhere to use plugin name like `::Patreon::PLUGINNAME`. So no more duplication.

Reply:
:+1: